app/scripts/views/reset_password.js

```diff
@@ -27,22 +30,13 @@ define(function (require, exports, module) {
       // address clears the email field in the formPrefill model if
       // the user doesn't enter an address. See comment in beforeDestroy
       this._formPrefill = options.formPrefill;
-
-      this.events = {
-        'click .remember-password': preventDefaultThen('_rememberPassword')
-      };
-
-      super.initialize(options);
```

Comment:
To future proof ourselves, I think there should be a `FormView.prototype.initialize.call(this, options);` here.